data/data/azure/vnet/nsg.tf

```
@@ -1,3 +1,6 @@
+locals {
+  description = "Created By OpenShift Installer"
+}
```

Separate with an empty line. Repeat for the other files where you added the `locals` block.
```suggestion
}

```